packages/trader/src/AppV2/Components/TradeParameters/LastDigitPrediction/last-digit-prediction.tsx

```diff
@@ -31,7 +31,6 @@ const LastDigitPrediction = observer(({ is_minimized, is_stats_mode }: TLastDigi
     };
     const onActionSheetClose = () => {
         setIsOpen(false);
-        //TODO: check if we need these 2 resets below after latest Quill Action sheet changes will be in our branch
```

Comment:
I checked, it's not connected with quill, it's because state storing in the parent. Hence, remove